tinyagentos/routes/coding_sessions.py

```
@@ -0,0 +1,280 @@
+                session["id"],
+                exc_info=True,
+            )
+
+    return session
+
+
+@router.get("/api/coding-sessions")
+async def list_coding_sessions(
+    request: Request,
+    include_archived: bool = False,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.18)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)